service_configs/vulnerability_assessment.yml

```diff
@@ -0,0 +1,11 @@
+bugzilla:
+    url: "https://bugzilla.mozilla.org/rest/"
+    creator: "infosec+rra2json@mozilla.com"
+    rra:
```

Comment:
If config use the same code, lets change that from `rra` to, well, just put it in `bugzilla` directly.

Alternative:

```
bugzilla:
....

    bugs:
      - rra:
          product: ..
          ...
      - va:
         product: ...
...
```

Reply:
I was going a slightly different route, mainly because then you'll need to expose switches to say what you want, unless the default is to run both in a single run, in which case, I'm cool with that approach.